src/Altinn.App.Core/Internal/Pdf/PdfOptionsMapping.cs

```diff
@@ -153,7 +153,10 @@ private static List<string> GetMappingValues(JObject jsonData, KeyValuePair<stri
             }
             else
             {
-                selectedDatas.Add(jsonData.SelectToken(select).ToString());
+                JToken? token = jsonData.SelectToken(select);
+                if (token != null) {
+                    selectedDatas.Add(token.ToString());
```

Comment:
Not related to this PR, but I'm looking forward to the day someone discovers that `.ToString()` on a number is dependent on the thread locale settings to figure out if the number is `0.1` or `0,1`

Reply:
Been there! :cry: I once spent an entire day trying to figure out why a calendar application wasn't rendering at all. No error messages, no hints. Dissecting commits revealed the cause was one commit (where the locale changed), and the cause of the rendering bug was some dynamically generated CSS that now contained commas instead of periods. Took me quite a long time squinting to find those pixels that marks the difference between a comma and a period. :joy: Learned a lot that day.